ci-operator/jobs/openshift/console/openshift-console-master-postsubmits.yaml

```diff
@@ -17,6 +17,7 @@ postsubmits:
         - --give-pr-author-access-to-namespace=true
         - --promote
         - --target=[images]
+        - --target=console-tests
```

Comment:
you should not need this -- the fact that promotion requires the image should build the dependency graph appropriately

Reply:
I just re-ran the job generation when CI complained. If I remove this, won't CI complain about not having run the job gen?